Corrected 3.3 release notes regarding registry

miminar opened the pull request:
- Schema 2 is supported by default.
- Enable accepting schema 2 on pushes only in case where backwards
  compatibility is not needed.
- Manifest lists are not yet supported.

Signed-off-by: Michal Minář miminar@redhat.com

miminar commented:
@adellape can you please check my wording?

ncdc commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
 
-To preserve compatibility with older docker versions, support for schema 2 must
-be manually enabled:
+Support for consuming schema 2 from external resources is enabled implicitely.
```
typo: implicitly

ncdc commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
 $ oc set env dc/docker-registry -n default REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2=true
 ----
 
+Manifest lists, providing support for Multi-architecture images, inctroduced by
```
typo: introduced

adellape commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -151,10 +151,9 @@ parameter] to define the name of the GCS bucket to store objects in.
-registry 2.4, and the features will be backported to {product-title} 3.2.
-Version 2.4 of the registry includes a variety of performance and usability
-enhancements, notably:
+The {product-title} 3.3 registry is based on docker distribution registry 2.4,
```
Thinking now that we should init cap this (and the section title) as "Docker Distribution", no? From looking around at capitalization in https://github.com/docker/distribution.

miminar commented:
Thanks! Fixed.
On 11.10.2016 15:13, Andy Goldstein wrote:
>
> *@ncdc* commented on this pull request.
>
> ------------------------------------------------------------------------------------------------------------------------
>
> In release_notes/ocp_3_3_release_notes.adoc 
> <https://github.com/openshift/openshift-docs/pull/3029#pullrequestreview-3670634>:
>
> >
> -To preserve compatibility with older docker versions, support for schema 2 must
> -be manually enabled:
> +Support for consuming schema 2 from external resources is enabled implicitely.
>
> typo: implicitly
>
> ------------------------------------------------------------------------------------------------------------------------
>
> In release_notes/ocp_3_3_release_notes.adoc 
> <https://github.com/openshift/openshift-docs/pull/3029#pullrequestreview-3670634>:
>
> >
>   ----
>   $ oc login -u system:admin
>   $ oc set env dc/docker-registry -n default REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2=true
>   ----
>   
> +Manifest lists, providing support for Multi-architecture images, inctroduced by
>
> typo: introduced
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/openshift/openshift-docs/pull/3029#pullrequestreview-3670634>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABaQk7iRGAcVsTEJclWKIq1S17iCWWFXks5qy4uIgaJpZM4KTmWC>.
>

adellape commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
-versions of an image (e.g., `amd64` versus `ppc64le`). Schema 2 also supports
-the ability to hash an image's configuration, to create an ID for the image and
-provide docker content-addressable information about the image.
+The image manifest version 2, schema2 introduces the ability to hash an image's
```
Either keep "schema2" wrapped in commas like it was, or put it in parentheses.

adellape commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
-the ability to hash an image's configuration, to create an ID for the image and
-provide docker content-addressable information about the image.
+The image manifest version 2, schema2 introduces the ability to hash an image's
+configuration and thus reduce manifest size, to create an ID for the image
```
Comma after "image".

adellape commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
-provide docker content-addressable information about the image.
+The image manifest version 2, schema2 introduces the ability to hash an image's
+configuration and thus reduce manifest size, to create an ID for the image
+and provide docker content-addressable information about the image.
```
"provide docker with"

miminar replied:
I will remove the "docker" instead.

adellape commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
 
-To preserve compatibility with older docker versions, support for schema 2 must
-be manually enabled:
+Support for consuming schema 2 from external resources is enabled implicitly.
```
"schema2" or "schema 2"? We should pick one and stick with it.

miminar replied:
Using `schema 2` consistently now.

adellape commented on release_notes/ocp_3_3_release_notes.adoc:
```diff
@@ -173,20 +172,24 @@ secondary source repository.
 $ oc set env dc/docker-registry -n default REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2=true
 ----
 
+Manifest lists, providing support for Multi-architecture images, introduced by
```
s/Multi/multi/

adellape replied:
"Manifest lists, introduced by schema 2 to provide support for multi-architecture images (e.g., `amd64` versus `ppc64le`), are not yet supported in {product-title} 3.3."

miminar replied:
Fixed.

miminar commented:
@adellape were your concerns addressed?

ahardin-rh commented:
@miminar @adellape Resurrecting this oldie. :ghost: Is this still valid, or should it be closed?

miminar commented:
> @miminar @adellape Resurrecting this oldie. :ghost: Is this still valid, or should it be closed?

It's still valid. The impact is low though.

ahardin-rh commented:
Thanks! :stars:

ahardin-rh commented:
No revision history needed
